core/src/gl/error.cpp

```diff
@@ -26,4 +26,14 @@ bool Error::hadGlError(const std::string& _locationTag) {
     return false;
 }
 
+void Error::glError(const char* stmt, const char* fname, int line) {
+    GLenum err = glGetError();
```

Comment:
`glGetError` should always be called in a loop because multiple error flags can be set at the same time (even by one function) and they're returned in an arbitrary order (http://docs.gl/es2/glGetError).